interface/utils.py

```diff
@@ -0,0 +1,9 @@
+from datetime import date, datetime
+
+
+def json_serial(obj):
+    """JSON serializer for objects not serializable by default json code"""
+
+    if isinstance(obj, (datetime, date)):
```

Comment:
Aaaaaaactually, isn't there one of these in Django?

Reply:
Yes, I found out that `DjangoJSONEncoder` also works here :)